### DISCUSSION TITLE
Failed to send metrics: Unable to send datapoints post Helm upgrade from 0.72.0 to latest version [Help wanted]

### DISCUSSION THREAD

**Discussion by sathishrpt24** [signalfx/splunk-otel-collector-chart#1456] [General]
We have a otel collector that is running 0.72.0 hem chart version without any issues. But when we upgrade helm chart to latest version noticed we are getting WARN "com.splunk.javaagent.shaded.io.micrometer.signalfx.SignalFxMeterRegistry - failed to send metrics: Unable to send datapoints" in the logs.

The issue is similar to what specified here - https://github.com/signalfx/splunk-otel-collector-chart/issues/766

Please help to fix this issue.

**Comment by atoulme** [comment]
Thank you for your report, please file a support ticket to receive timely help: https://splunk.my.site.com/customer/s/need-help/create-case
